src/query/settings/src/settings_default.rs

```diff
@@ -590,6 +590,15 @@ impl DefaultSettings {
                     mode: SettingMode::Both,
                     range: Some(SettingRange::Numeric(0..=1)),
                 }),
+                (
```

Comment:
Maybe just use enable_compact_after_write, no need to add new settings.

Reply:
Multi-table insert is special. The hook needs to compact and recluster multiple tables. To avoid potential performance degradation, a seperate setting is introduced.